docs/sources/_index.md

```diff
@@ -8,14 +8,12 @@ weight: 100
 
 # Grafana Loki documentation
 
-<p align="center"> <img src="logo_and_name.png" alt="Loki Logo"> <br>
-  <small>Like Prometheus, but for logs!</small> </p>
+<p align="center"> <img src="logo_and_name.png" alt="Loki Logo"> <br> 
 
-Grafana Loki is a set of components that can be composed into a fully featured
-logging stack.
+Grafana Loki is a set of components that can be composed into a fully featured logging stack.
 
-Unlike other logging systems, Loki is built around the idea of only indexing
-metadata about your logs: labels (just like Prometheus labels). Log data itself
-is then compressed and stored in chunks in object stores such as S3 or GCS, or
-even locally on the filesystem. A small index and highly compressed chunks
-simplifies the operation and significantly lowers the cost of Loki.
+Unlike other logging systems, Loki is built around the idea of only indexing metadata about your logs: labels (just like Prometheus labels). 
+Log data itself is then compressed and stored in chunks in object stores such as S3 or GCS, or even locally on the filesystem. 
```

Comment:
nit: Maybe more of a personal preference, but calling out the service names by their full name sounds more professional:

```suggestion
Log data itself is then compressed and stored in chunks in object stores such as Amazon Simple Storage Service (S3) or Google Cloud Storage (GCS), or even locally on the filesystem. 
```

Or at least:


```suggestion
Log data itself is then compressed and stored in chunks in object stores such as Amazon's S3 or Google's GCS, or even locally on the filesystem. 
```

I guess this would be something to change throughout our docs, so not really relevant for this PR. However, I'm curious what you think.

Reply:
Not a nit at all, I agree.  I'd actually made that update, not sure why it didn't make it into the PR.